k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -80,6 +80,14 @@ const (
 	UnmapMarkSnapChainRemovedEnabled  = UnmapMarkSnapChainRemoved("enabled")
 )
 
+type VolumeOfflineRebuild string
```

Comment:
The global setting is `OfflineReplicaRebuilding`. Make the naming consistent.

```suggestion
type VolumeOfflineRebuilding string
```

Reply:
Modified. Thanks